Templates/local_openshift/rebuild_fluentd.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+oc delete all -l template=hogarama-fluentd
+oc delete persistentvolumeclaim -l template=hogarama-fluentd
+oc delete secrets -l template=hogarama-fluentd
+oc process -f hogarama-fluentd.yaml BRANCH=master | oc create -f -
```

Comment:
maybe use $(git branch | grep \* | cut -d ' ' -f2)  to use current branch instead of master.